Engine/src/Console/CVarUtil.bf

```diff
@@ -0,0 +1,182 @@
+using System;
+
+namespace SteelEngine.Console
+{
+	static class CVarUtil
+	{
+		public static bool TryParse(CVar cvar, Span<StringView> args, ref bool result, out bool didChange)
+		{
+			let currentVal = result;
+			switch (args[0])
+			{
+			case "true", "True", "TRUE", "1":
+				result = true;
+			case "false", "False", "FALSE", "0":
```

Comment:
Instead of checking for the different spellings, why not convert these to lowercase and then just check for ["false", 0] and ["true", 1]?